tests/log-in.test.js

```diff
@@ -0,0 +1,75 @@
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+describe('Banking', () => {
+
+    it('Debería ingresar a la pagina con datos correctos', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
+
+        let pssWord = await $('#password');
+        await pssWord.setValue('Demo123!');
+
+        let submit = await $('#submit');
+        await submit.click();
+
+        let url = await browser.getUrl(); 
+        //await console.log(url);
+        //await console.log("salida");
+
+        await expect(url).to.equal('http://localhost:8080/bank/home');
+
+        //await browser.pause(5000);
+    });
+    
+    it('Debería recordar sesión iniciada', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
+
+        let pssWord = await $('#password');
+        await pssWord.setValue('Demo123!');
+
+        let rememberMe = await $('#remember-me');
+        await rememberMe.click();
+
+        let submit = await $('#submit');
+        await submit.click();
+
+        await browser.newWindow('http://localhost:8080/bank/');
+        await expect(await browser.getUrl()).to.equal('http://localhost:8080/bank/home');
+
+        //await browser.pause(5000);
+    });
+
+    it('No deberia ingresar con credenciales erroneas', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
+
+        let pssWord = await $('#password');
+        await pssWord.setValue('QWERTY123');
+
+        let enviar = await $('#submit');
+        await enviar.click();
+
+        await expect(await browser.getUrl()).to.equal('http://localhost:8080/bank/login?error');
```

Comment:
Prefer evidence that's more related to the state of the page for this one, as the URL is not tipically something directly visible for the end user. There is usually some clear visual cue inside the page's structure itself that gives direct feedback about the error. Prefer that one, as it's more robust and user oriented.